README.md

```diff
@@ -10,7 +10,11 @@ This repository contains Docker imgaes for different flavours of Infinispan:
 
 Infinispan can be used in two different ways: in client-server mode and in embedded mode.  In client-server mode Infinispan runs as standalone server in diferent JVM and application using Infinispan connects to the server as a client. On the other hand, in embedded mode Infinispan runs in the same JVM as application and application access it as a library. For more details see [Infinispan documentation](http://infinispan.org/docs/7.0.x/user_guide/user_guide.html#_server_modules).
 
+## Branching strategy
 
+[Master branch](https://github.com/jboss-dockerfiles/infinispan/tree/master) contains current development version. Each major stable version has its own branch (e.g. [8.2.x](https://github.com/jboss-dockerfiles/infinispan/tree/8.2.x)). All minor versions are tagged from that branch (e.g. 8.2.1.Final, 8.2.2.final).
```

Comment:
As long as "8.2.1.Final" is for you minor. I would call it micro :) Same goes to "major" above.